velox/exec/RowNumber.cpp

```diff
@@ -262,7 +241,11 @@ FlatVector<int64_t>& RowNumber::getOrCreateRowNumberVector(vector_size_t size) {
 
 RowVectorPtr RowNumber::getOutput() {
   if (input_ == nullptr) {
```

Comment:
```
  if (input_ == nullptr) {
    if (spillInputReader_ != nullptr) {
      recursiveSpillInput();
    }
    if (input_ == nullptr)
      return nullptr;
    }
  }
```
Can we guarantee there is always data to restore here? Shall we add to check input_ after recursive spill input? thanks!